src/main/java/com/dreammaster/gthandler/recipes/AlloySmelterRecipes.java

```diff
@@ -8,236 +11,343 @@
 import com.dreammaster.gthandler.CustomItemList;
 
 import cpw.mods.fml.common.Loader;
-import gregtech.api.enums.GT_Values;
-import gregtech.api.enums.ItemList;
-import gregtech.api.enums.Materials;
-import gregtech.api.enums.OrePrefixes;
+import gregtech.api.enums.*;
 import gregtech.api.util.GT_ModHandler;
 import gregtech.api.util.GT_OreDictUnificator;
 import gregtech.api.util.GT_Utility;
 
 public class AlloySmelterRecipes implements Runnable {
 
+    final String modIDEnderIO = "EnderIO";
+    final String modIDTinkerConstruct = "TConstruct";
+
     @Override
     public void run() {
+        registerCoreModRecipes();
+        registerEnderIORecipes();
+        registerTinkerConstructRecipes();
+
+    }
+
+    public void registerTinkerConstructRecipes() {
+        if (!Loader.isModLoaded(modIDTinkerConstruct)) {
+            return;
+        }
+
+        ItemStack x3_AluminiumDusts = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 3L);
+
+        ItemStack x1_CopperDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Copper, 1L);
+
+        ItemStack x4_AluminiumBrassIngots = GT_ModHandler.getModItem(modIDTinkerConstruct, "materials", 4L, 14);
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x3_AluminiumDusts,
+                x1_CopperDust,
+                x4_AluminiumBrassIngots,
+                5 * SECONDS,
+                (int) TierEU.RECIPE_MV);
+
+    }
+
+    public void registerCoreModRecipes() {
+        ItemStack x4_MicaBasedSheets = CustomItemList.MicaBasedSheet.get(4L);
+
+        ItemStack x1_SiliconDioxideDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L);
+
+        ItemStack x4_MicaInsulatorSheets = CustomItemList.MicaInsulatorSheet.get(4L);
+
+        ItemStack x1_RubyDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Ruby, 1L);
+
+        ItemStack x1_SapphireDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sapphire, 1L);
+
+        ItemStack x1_GreenSapphire = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_NetherQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_CertusQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1L);
+
+        ItemStack x2_AluminoSilicateWools = CustomItemList.AluminoSilicateWool.get(2L);
```

Comment:
I'd drop the -s suffix for plurals, it's error prone when copy-pasting x1 stuff

Reply:
agreed